spec/models/guest_spec.rb

```diff
@@ -0,0 +1,15 @@
+require 'rails_helper'
+
+RSpec.describe Guest, type: :model do
+ it { should have_many :rooms }
+
+  describe ".scope" do 
+    it '.count_guests' do 
+      Guest.create!(name: "Hannah", nights: 3)
+      Guest.create!(name: "Phil", nights: 3)
+      Guest.create!(name: "Denise", nights: 3)
+
+      expect(Guest.count_guests).to eq(3)
```

Comment:
Just by looking at this current test, there really isn't a way for us to know whether this call to `count_guests` is actually running some logic or if it contains a hard-coded `3` in the method. We should add another Guest after the first expect and check the count again to see it update. This way we are sure that it's the logic making the test pass.

```ruby
expect(Guest.count_guests).to eq(3)

Guest.create!(name: "Denise", nights: 3)

expect(Guest.count_guests).to eq(4)
```

If we think about the TDD process, it says "write the amount piece of code to get the test to pass", in your assertion, the smallest amount would have been to just hard-code `3`, so we need to make the test more robust.